clientd3d/server.c

```diff
@@ -389,7 +365,7 @@ void ExtractObjectNoLight(char **ptr, object_node *item)
 object_node *ExtractNewObject(char **ptr)
 {
    object_node *item = ObjectGetBlank();
-   ExtractObject(ptr, item);
+   ExtractObject(ptr, item, true);
```

Comment:
You don't need to put true everywhere.. that's what the default value is for.

Reply:
Changed all references  to remove the `true` parameter after adjusting the default value in `server.h`

Also used `True` instead of `true` since the capitalized version is what is used throughout the rest of the `server.c` file.